### ISSUE TITLE
Segmentation fault when all the dates are imprecise

### ISSUE THREAD

**Issue by annazhukova** [tothuhien/lsd2#10]
Hi Hien,

I have a data set where the only information available for the sampling is year, and I tried to specify the dates (all of them) as intervals, e.g. b(1970.0,1970.9972602739726) for 1970, then to run LSD2 (v1.4.2.2) as 
```bash
lsd2 -i {input_tree} -d {input_dates} -v 2 -c -s {sequence_length} -f 1000 -o {work_dir} -e 3
```
What I get is a segmentation fault:
```bash
TREE 1
*PROCESSING:
Reading the tree ... 
Calculating the outlier nodes ...
bash: line 2: 13495 Segmentation fault      (core dumped) 
```
I suspect it might be due to the fact that intervals are not taken into account for outlier detection. In any case at least a more informative error message would be helpful.

Thanks,
Anna

**Comment by tothuhien** [comment]
Hi Anna,
Yes you're right, I didn't consider to put an error message in this case. I've fixed it in the latest commit. Will make a new release when I fix the other issue #11  as well. Thanks a lot for your feedback.
Hien

**Comment by annazhukova** [comment]
Thanks Hien!
